app/models/institution.rb

```diff
@@ -159,6 +159,11 @@ class Institution < ActiveRecord::Base
   validates :institution_type_name, inclusion: { in: TYPES }
 
   has_many :yellow_ribbon_programs, dependent: :destroy
+  has_many :school_certifying_officials, dependent: :destroy
```

Comment:
Please use https://edgeguides.rubyonrails.org/association_basics.html#scopes-for-has-many-order so that the ordering function can be removed from https://github.com/department-of-veterans-affairs/vets-website/pull/10690

https://stackoverflow.com/a/19138677/1669481

```suggestion
  has_many :school_certifying_officials, -> { order 'priority, lastName' }, dependent: :destroy
```